clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -394,6 +410,33 @@ export class SettingsService {
       .get(section);
     return SettingsService.evaluateVariables(pathList, vars);
   }
+
+  /**
+   * Gets unreachable code diagnostics severity
+   * @returns Error, Warning, Information for ERROR, WARN, INFO, HINT or undefined for all other cases
+   */
+  public static getUnreachableCodeSeverity():
+    | vscode.DiagnosticSeverity
+    | undefined {
+    const value = SettingsService.getUnreachableCodeSeverityValue();
```

Comment:
It is used just here.
```suggestion
    const value = scode.workspace
        .getConfiguration()
        .get(SETTINGS_UNREACHABLE_CODE_SEVERITY) ?? NONE;
```

Reply:
Fixed